packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -254,7 +254,7 @@ export function sanitizeTokenSymbol(
   options: SanitizeTokenOptions
 ) {
   if (!options.erc20L1Address) {
-    return tokenSymbol.toUpperCase()
+    return tokenSymbol
```

Comment:
This change removes token names being force upper-cased in token lists. Eg. `3Crv` -> `3CRV`